cmd/kube-apiserver/app/server.go

```diff
@@ -556,7 +565,9 @@ func (s *APIServer) Run(_ []string) error {
 		Authorizer:                authorizer,
 		AdmissionControl:          admissionController,
 		DisableV1:                 disableV1,
-		EnableExp:                 enableExp,
+		EnableExtensions:          enableExtensions,
```

Comment:
thanks, that'd be great. I was thinking not so much of the v1 group, but
the new groups we'll make in the future.

On Tue, Oct 13, 2015 at 1:45 PM, Nikhil Jindal <notifications@github.com>
wrote:

> In cmd/kube-apiserver/app/server.go
> <https://github.com/kubernetes/kubernetes/pull/15516#discussion_r41922410>
> :
>
> > @@ -556,7 +565,9 @@ func (s *APIServer) Run(_ []string) error {
> >  		Authorizer:                authorizer,
> >  		AdmissionControl:          admissionController,
> >  		DisableV1:                 disableV1,
> > -		EnableExp:                 enableExp,
> > +		EnableExtensions:          enableExtensions,
>
> Yes I can make this a generic struct:
>
> ResourceOverrides struct {
>   groupVersion string,
>   resources util.Set,
>   isEnableList bool
> }
>
> I was thinking of allowing overrides only for the extensions group and not
> v1 and that is why I did not put the generic struct, but will do it now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15516/files#r41922410>.
>